tensorrt_llm/_torch/pyexecutor/config.py

```diff
@@ -50,6 +50,11 @@ class PyTorchConfig:
     attention_dp_time_out_iters: int = 50
     attention_dp_batching_wait_iters: int = 10
 
+    attention_tp_enable_waiting: bool = False
```

Comment:
This seems not right. Its not "attention_tp", Its just TP right?

How about "enable_batch_delay"?

Reply:
or "enable_batch_wait", since line 58 already has `batch_wait_timeout_ms`